config/application.rb

```diff
@@ -6,7 +6,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-module ItemRepository
+module Mothership
   class Application < Rails::Application
```

Comment:
Nice catch, I wonder why this didn't throw errors previously?

Reply:
I think this module is not actually referenced anywhere.